axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -620,6 +620,14 @@
         "initializeArgs": {
           "gasCollector": "GALITAYSFTLEVXIBTF5FVUIV7PFVF7VF5VSZVFYCXA7PXSZ2LZMY54VG"
         }
+      },
+      "example": {
+        "address": "CDV63R2YDFEYWXM5S4C3MLW65SIOKZYAY67NUGJ4QOGJXK6LJOSZAWLJ",
```

Comment:
old or new deployment?

Reply:
this is a new deployment with the latest code.